module2/homework/transformContainers/uniquemap.cpp

```diff
@@ -0,0 +1,25 @@
+#include "uniquemap.hpp"
+
+#include <algorithm>
+#include <iterator>
+#include <utility>
+
+std::map<int, std::string> removeDuplicateAndTranformToMap(std::list<std::string>& inputList,
```

Comment:
W sumie to bardziej pytanie niż uwaga :) Wiecie może jaka jest konwencja nazywania zmiennych w C++? Czy takim camelcase (inputList) czy z podkreśleniem (input_list)? W sumie to pewnie zależy od autora

Reply:
Z tego co wiem to w Pythonie używa się snake case (z podkreśleniem), w C++ niektórzy twierdzą, że camelCase, ale to pewnie zależy od projektu i ludzi, którzy go robią ;) w pracy my się musimy do wszystkiego dostosować, do formatowania też :D

Reply:
Ja mam właśnie takie z Pythona_ przyzwyczajenie dlatego się zastanawiałem ;)